registry/pluginloader/pluginloader.go

```diff
@@ -0,0 +1,18 @@
+// +build go1.8
+
+package pluginloader
+
+import (
+	"path/filepath"
+	"plugin"
+)
+
+// NOTE: should suffix be a platform specific? Like dylib on darwin
```

Comment:
I prefer using the platform specific dynamic library suffix. Below you also mention `.so`.